docs/resources/secmaster_incident.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "SecMaster"
+---
+
+# huaweicloud_secmaster_incident
+
+Manages a SecMaster incident resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "workspace_id" {}
+
+resource "huaweicloud_secmaster_incident" "test" {
+  workspace_id = var.workspace_id
+  name         = "test-incident"
+  description  = "test incident"
+
+  type {
+    category      = "DDoS"
+    incident_type = "ACK Flood"
+  }
+
+  level  = "Tips"
+  status = "Open"
+  owner  = "test-user"
+
+  data_source {
+    product_feature = "hss"
+    product_name    = "hss"
+    source_type     = 1
+  }
+
+  first_occurrence_time = "2023-04-18T13:00:00.000+08:00"
+  last_occurrence_time  = "2023-04-19T14:00:00.000+08:00"
+  planned_closure_time  = "2023-04-20T15:00:00.000+08:00"
+  verification_status   = "Unknown"
+  stage                 = "Preparation"
+  debugging_data        = false
+  labels                = "test1,test2"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the ID of the workspace to which the incident belongs.
+
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the incident name.
+
+* `description` - (Required, String) Specifies the incident description.
+
+* `type` - (Required, List, ForceNew) Specifies the incident type configuration.
+  The [IncidentType](#Incident_IncidentType) structure is documented below.
+
+  Changing this parameter will create a new resource
+
+* `level` - (Required, String) Specifies the incident level.
+  The value can be **Tips**, **Low**, **Medium**, **High** and **Fatal**.
+
+* `status` - (Required, String) Specifies the incident status.
+  The value can be **Open**, **Block** and **Closed**.
+
+* `data_source` - (Required, List, ForceNew) Specifies the data source configuration.
+  The [IncidentDataSource](#IncidentDataSource) structure is documented below.
+
+  Changing this parameter will create a new resource
+
+* `first_occurrence_time` - (Required, String) Specifies the first occurrence time of the incident.
```

Comment:
Please add an example for time filed.
`Specifies the first occurrence time in format **XXX** of the incident, e.g. **XXX**.`

Reply:
not easy to express the format, but I add a example